app/errors.py

```diff
@@ -0,0 +1,13 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass

```

Comment:
Consider adding a constructor to the `NotVaccinatedError` class to accept a custom error message. This will provide more context when the exception is raised.